### DISCUSSION TITLE
Sidebar in user blueprint

### DISCUSSION THREAD

**Discussion by Sennik** [statamic/cms#5148] [Q&A] [answered]
I've added a section named sidebar, handle sidebar in the user blueprint but it still shows up as a tab. Is it not possible to add a sidebar here?

**Comment by jackmcdade** [comment]
How wide is your screen? It needs to be at least `1210px` wide for the sidebar to show up.

**Reply by Sennik** [reply]
1920x1200, I have sidebar in the other collections, it just does not show up in the user screen

**Comment by jackmcdade** [comment, marked as answer]
Oh i don't think Users supports a sidebar, unfortunately.

**Reply by Sennik** [reply]
Ok, my bad ;-) Shall I make a suggestions in the doc's to mention that?

**Comment by jackmcdade** [comment]
It's a bit more of an oversight than a feature – maybe a FR on statamic/ideas?
